lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -116,6 +124,80 @@ function preflightCheck(
     }
 }
 
+/**
+ * Public method to indicate whether platform broker is available to make native token request.
+ */
+export async function isPlatformBrokerAvailable(
+    logger?: LoggerOptions,
+    performanceClient?: IPerformanceClient
+): Promise<boolean> {
+    const defaultLoggerOptions: LoggerOptions = {
+        loggerCallback: (): void => {
+            // Empty logger callback
+        },
+        piiLoggingEnabled: false,
+        logLevel: LogLevel.Trace,
+    };
+
+    const temporaryLogger = new Logger(
+        logger || defaultLoggerOptions,
+        name,
+        version
+    ); // Default logger
+
+    const defaultPerformanceClientConfig = {
+        auth: {
+            clientId: "test-clientid",
+        },
+    };
+    const temporaryTelemetryClient: IPerformanceClient =
+        performanceClient ||
+        new BrowserPerformanceClient(defaultPerformanceClientConfig);
+
+    temporaryLogger.trace("isPlatformBrokerAvailable called");
+
+    // Check if DOM platform API is supported
+
+    // @ts-ignore
+    if (window.navigator?.platformAuthentication) {
+        const supportedContracts =
+            // @ts-ignore
+            await window.navigator.platformAuthentication.getSupportedContracts(
+                "MicrosoftEntra"
```

Comment:
What is our latest on constants for min file optimizations? Do we directly use strings? If these are standardized, I would like a constant defined elsewhere.